coding-standard-baseline/action.yml

```diff
@@ -0,0 +1,118 @@
+name: "M2 Coding Standard"
+author: "Mage-OS"
```

Comment:
Can you add a test of this action against the fake mage-os package source just to prove that it works?

Reply:
Hi @damienwebdev ,  we have tested on local and below are results 

https://github.com/MeetKamal/mageos-magento2/actions/runs/5507076138/jobs/10036558694

Reply:
Let's move forward and merge this. If they do not work for whatever reason, a new PR needs to fix it.